lib/spoom/sorbet/sigils.rb

```diff
@@ -56,14 +56,14 @@ def self.update_sigil(content, new_strictness)
       sig { params(path: T.any(String, Pathname)).returns(T.nilable(String)) }
       def self.file_strictness(path)
         return nil unless File.exist?(path)
-        content = File.read(path)
+        content = File.read(path, encoding: Encoding::ASCII_8BIT.to_s)
```

Comment:
Super nitpicky but you really should not need the `.to_s` here.

Reply:
Good call: https://github.com/sorbet/sorbet/pull/3861

Reply:
Fixed 🎉